docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -33,7 +33,31 @@ Keeping informed
 To track changes that affect collections:
 
 * Join the `Collection Maintainers & Contributors forum group <https://forum.ansible.com/g/CollectionMaintainer>`_.
-* Subscribe to the `Bullhorn <https://forum.ansible.com/c/news/bullhorn/17>`_ Ansible contributor newsletter.
+* Subscribe to the `Bullhorn <https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn>`_ Ansible contributor newsletter.
+
+.. _coll_wg_reqs:
+
+Communication and Working Groups
```

Comment:
We talk about tags, then Groups without explaining what either are.

We need to be clear about when someone may want a group (rather just a tag)

Also detail how to request a tag

Reply:
SGTM. Added info, reorganized. thanks, PTAL
Though now i have concerns about the section verbosity or is it OK?